packages/integrations/preact/package.json

```diff
@@ -35,7 +35,8 @@
     "@babel/core": ">=7.0.0-0 <8.0.0",
     "@babel/plugin-transform-react-jsx": "^7.17.12",
     "babel-plugin-module-resolver": "^4.1.0",
-    "preact-render-to-string": "^5.2.0"
+    "preact-render-to-string": "^5.2.0",
+    "@preact/signals": "1.0.3"
```

Comment:
This is a dep and not a peerDep so that it doesn't require users to use preact/signals.